x-pack/test_serverless/shared/services/svl_reporting.ts

```diff
@@ -37,29 +36,35 @@ export function SvlReportingServiceProvider({ getService }: FtrProviderContext)
       const requestPath = `${INTERNAL_ROUTES.GENERATE_PREFIX}/${jobType}`;
       log.debug(`POST request to ${requestPath}`);
 
-      const { status, body } = await supertestWithoutAuth
+      const { body }: { status: number; body: ReportingJobResponse } = await supertestWithoutAuth
         .post(requestPath)
         .set(internalReqHeader)
         .set(roleAuthc.apiKeyHeader)
```

Comment:
If we call here internal API, we should also use `cookieHeader` instead of API key.
More details in the [docs](https://github.com/elastic/kibana/tree/main/x-pack/test_serverless#api-integration-test-example)